modules/@apostrophecms/template/index.js

```diff
@@ -673,6 +673,22 @@ module.exports = {
           req.statusCode = 500;
           return self.render(req, 'templateError');
         }
+
+        function unrefreshed(url) {
+          // Including apos-refresh=1 in data.url leads to busted pages in
+          // navigation links, so strip that out. However this is invoked on
+          // every page load so do it as quickly as we can to avoid the
+          // overhead of a full parse and rebuild
+          if (!url.includes('apos-refresh=1')) {
```

Comment:
I tested this solution with this:

```
console.log(unrefreshed('/wiggle/wiggly?apos-refresh=1'));
console.log(unrefreshed('/wiggle/wiggly?apos-refresh=1&coolness=true'));
console.log(unrefreshed('/wiggle/wiggly?coolness=true&apos-refresh=1'));
console.log(unrefreshed('/wiggle/wiggly?coolness=true&apos-refresh=1&badness=false'));
        function unrefreshed(url) {
          // Including apos-refresh=1 in data.url leads to busted pages in
          // navigation links, so strip that out. However this is invoked on
          // every page load so do it as quickly as we can to avoid the
          // overhead of a full parse and rebuild
          if (url.endsWith('?apos-refresh=1')) {
            return url.replace('?apos-refresh=1', '');
          } else if (url.includes('?apos-refresh=1')) {
            return url.replace('?apos-refresh=1&', '?');
          } else {
            return url.replace('&apos-refresh=1', '');
          }
        }
```